src/Components/Pages/Pricing.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import Navbar from "../Layout/NavBar";
+import {
+  Button,
+  Container,
+  Card,
+  CardDeck,
+  ListGroup,
+  ListGroupItem,
+} from "react-bootstrap";
+
+export class Pricing extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tiers: [
+        {
+          name: "Free",
+          price: "$0/mo",
+          perks: [
+            "Up to 5 Workshop Levels",
+            "Limited to 20 students",
+            "No concurrent workshops",
+          ],
+        },
+        {
+          name: "Pro",
+          price: "$10/mo",
+          perks: [
+            "Up to 25 Workshop Levels",
+            "Limited to 100 students",
+            "5 concurrent workshops",
+          ],
+        },
+        {
+          name: "Enterprise",
+          price: "$50/mo",
+          perks: [
+            "Unlimited Workshop Levels",
+            "Unlimited students",
+            "Unlimited concurrent workshops",
+          ],
+        },
+      ],
+    };
+  }
+
+  render() {
+    let tierCards = this.state.tiers.map((item) => (
+      <Card style={{ width: "18rem" }}>
+        <Card.Img variant="top" />
+        <Card.Body>
+          <Card.Title>{item.name}</Card.Title>
+          <Card.Text>{item.price}</Card.Text>
+        </Card.Body>
+        <ListGroup className="list-group-flush">
+          <ListGroupItem>{item.perks[0]}</ListGroupItem>
+          <ListGroupItem>{item.perks[1]}</ListGroupItem>
+          <ListGroupItem>{item.perks[2]}</ListGroupItem>
+        </ListGroup>
+        <Button>Sign Up!</Button>
+      </Card>
+    ));
+
+    return (
+      <div>
+        <Navbar />
+        <Container fluid>
+          <div className="m-5 p-5 floating-icon">
+            <h1>Pricing</h1>
+            <p>
```

Comment:
Perhaps add an mt-3 to the <p> tag? Bit more spacing between heading and description